charts/agent/templates/daemonset.yaml

```diff
@@ -244,6 +244,9 @@ spec:
           {{- /* Always requested */}}
             - mountPath: /host/dev
               name: dev-vol
+            - mountPath: /host/usr
```

Comment:
Since this volume was added for autopilot in https://github.com/sysdiglabs/charts/pull/1625/files#diff-18888d7372d81c06ea1ec38c2606ae16bf04d0248bdb8ad475cbc5cc22bbc364R523 i do expect to not be an issue to always mount it from now on